.augment/env/setup.sh

```
@@ -0,0 +1,49 @@
+echo "✅ npm installed: $npm_version"
+
+# Install build essentials for native modules (serialport, etc.)
+echo "🔧 Installing build tools for native modules..."
+sudo apt-get install -y build-essential python3 python3-pip
+
+# Install additional dependencies for Electron
+echo "🖥️ Installing Electron dependencies..."
+sudo apt-get install -y libnss3-dev libatk-bridge2.0-dev libdrm2 libxcomposite1 libxdamage1 libxrandr2 libgbm1 libxss1 libasound2
+
+# Navigate to project directory
+cd /mnt/persist/workspace
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)